jquery.pjax.js

```diff
@@ -292,10 +292,16 @@ function pjax(options) {
       window.history.replaceState(pjax.state, container.title, container.url)
     }
 
+    // Only blur the focus if the focused element is within the container.
+    var blurFocus = $.contains(options.container, $(':focus'));
```

Comment:
1. Could you use `document.activeElement` instead of `$(':focus')`
2. We don't use semicolons